### DISCUSSION TITLE
"WalletConnect Certified" badge

### DISCUSSION THREAD

**Discussion by aminhdev** [WalletConnect/walletconnect-monorepo#5540] [Developer Support]
We read the docs for "WalletConnect Certified" here: https://walletconnect.network/blog/walletguide-and-walletconnect-certified-the-future-of-digital-wallets
But the docs just say like below

> For wallets keen to explore how you can get involved with WalletConnect Certified and uphold your pledge to making the new internet more usable, be sure to have an updated version of WalletKit integrated in your product.

I don't know after updating to the latest WalletKit, and after that, what should I do next to get the "WalletConnect Certified" badge? 
Anyone who has experience in this, please help. Thanks in advance!

FYI: We already have a wallet app listed in https://walletguide.walletconnect.network

**Comment by arein** [comment]
@aminhdev which wallet are you building? :) WalletConnect will soon share more details on how to become Certified

**Reply by aminhdev** [reply]
> @aminhdev which wallet are you building? :) WalletConnect will soon share more details on how to become Certified

Thanks for your quick response. We're building KEYRING PRO app. We already updated to the latest WalletKit, and will soon release a new app version this week.
